addons/hr_skills/__manifest__.py

```diff
@@ -53,10 +53,20 @@
             'hr_skills/static/src/scss/*.scss',
             'hr_skills/static/src/views/skills_list_renderer.js',
             'hr_skills/static/src/components/**/*',
+            'hr_skills/static/src/core/common/**/*',
         ],
         'web.assets_backend_lazy': [
             'hr_skills/static/src/views/skills_graph.js',
         ],
+        'im_livechat.assets_embed_core': [
+            'hr_skills/static/src/core/common/**/*',
+        ],
+        'mail.assets_public': [
+            'hr_skills/static/src/core/common/**/*',
+        ],
+        'portal.assets_chatter_helpers': [
+            'hr_skills/static/src/core/common/**/*',
+        ],
```

Comment:
The 'portal.assets_chatter_helpers' should actually be read as `'portal.assets_chatter'` and the `'portal.assets_chatter` should be read `'portal.assets_fronent_chatter`.

At some point the rename could make it clearer. Ok for now.